spec/spec_helper.rb

```diff
@@ -28,7 +28,13 @@ def call(_subject)
 Dir[Pathname(__dir__).join('shared/*.rb')].each(&method(:require))
 require_relative '../lib/spec/dry/types'
 
+Undefined = Dry::Core::Constants::Undefined
+
 RSpec.configure do |config|
+  config.before(:example, :maybe) do
```

Comment:
I have no idea why the order matters, but `specs` where failing with `Dry::Container::Error:
Nothing registered with the key "maybe.strict.string"`